tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -88,17 +96,29 @@ void InternalCommandsHandler::execute(InternalCommandsHandler::ExecutionRequests
                                    req.outActualReplySize,
                                    req.outReplicaSpecificInfoSize);
     } else {
+      bool isBlockAccumulationEnabled = (ReplicaConfig::instance().blockAccumulation &&
```

Comment:
Enabling/disabling block accumulation happens in `ReplicaImp`, not here. The only difference at the commands handler level should be in the number of requests in the queue. I don't believe it's good practice to rely on the config property at this level.